auth/providers/azure/graph/graph.go

```diff
@@ -203,7 +203,7 @@ func (u *UserInfo) getMemberGroupsUsingARCOboService(ctx context.Context, access
 	}
 
 	// the arc obo service does not support getting groups for applications
-	if claims[idtypClaim] != nil {
+	if claims[idtypClaim] != nil && !strings.EqualFold(claims[idtypClaim].(string), "user") {
```

Comment:
What if `claims[idtypClaim]` is not a string somehow ? Can you add a UT to make sure we don't panic ?

Reply:
the logic has been updated, PTAL